include/rnp/rnp.h

```diff
@@ -1348,6 +1353,26 @@ RNP_API rnp_result_t rnp_key_get_subkey_at(rnp_key_handle_t  key,
                                            size_t            idx,
                                            rnp_key_handle_t *subkey);
 
+/** Get default key for specified usage. Accepts primary key
+ *  and returns one of its subkeys suitable for desired usage.
+ *  May return the same primary key if it is suitable for requested
+ *  usage and flag RNP_KEY_SUBKEYS_ONLY is not set.
+ *  available.
+ *
+ *  @param primary_key handle of the primary key.
+ *  @param usage see rnp_op_generate_add_usage() function description.
```

Comment:
I would add here something like `Desired key usage, i.e. 'encrypt', 'sign', and so on. See rnp_op_generate_add_usage() for all possible values` so user may understand possible values from a single lookup.